app/components/transformations/appointment.rb

```diff
@@ -0,0 +1,41 @@
+module Transformations
```

Comment:
We should reflect somewhere in the namespace that this object is concerned with transforming Booking Bug appointment data.